exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,12 @@
 var name = "Daniel";
 var danielsRole = "mentor";
 
+
+if(danielsRole === "mentor"){
+  let  result = `Hi, I'm ${name} I'm a mentor`;
+   console.log(result);
+}
```

Comment:
It is working but you need to add other condition in exercise ('If Daniel is a student....')